### PR TITLE
Update performance-guidelines-best-practices-storage.md

### PR DESCRIPTION
**Pull request by ujpat**
Storing page file was mentioned in our old documentation changes. But the new practices doesnt mention that (https://github.com/Huachao/azure-content/blob/master/articles/virtual-machines/virtual-machines-sql-server-performance-best-practices.md#performance-best-practices-for-sql-server-in-azure-virtual-machines)

### CODE REVIEW

**Comment by ujpat** [comment]
@bluefooted for approval

**Comment by ujpat** [comment]
also mentioned at: https://azure.microsoft.com/en-us/blog/virtual-machines-best-practices-single-vms-temporary-storage-and-uploaded-disks/ but quite difficult for customers to find in the first glance.

**Comment by Jak-MS** [comment]
@bluefooted 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/performance-guidelines-best-practices-storage.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Many Azure VMs contain another disk type called the temporary disk (labeled as t
 
 The temporary storage drive isn't persisted to remote storage and therefore shouldn't store user database files, transaction log files, or anything that must be preserved.
 
-Place `tempdb` on the local temporary SSD `D:\` drive for SQL Server workloads unless consumption of local cache is a concern. If you're using a VM that [doesn't have a temporary disk](/azure/virtual-machines/azure-vms-no-temp-disk) then it's recommended to place `tempdb` on its own isolated disk or storage pool with caching set to read-only. To learn more, see [tempdb data caching policies](performance-guidelines-best-practices-storage.md#data-file-caching-policies).
+Place `tempdb` on the local temporary SSD `D:\` drive for SQL Server workloads unless consumption of local cache is a concern. If you're using a VM that [doesn't have a temporary disk](/azure/virtual-machines/azure-vms-no-temp-disk) then it's recommended to place `tempdb` on its own isolated disk or storage pool with caching set to read-only. To learn more, see [tempdb data caching policies](performance-guidelines-best-practices-storage.md#data-file-caching-policies). Only store TempDB and/or Buffer Pool Extensions/Page File on the D drive.
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/performance-guidelines-best-practices-storage.md]:
```suggestion
Place `tempdb` on the local temporary SSD `D:\` drive for SQL Server workloads unless consumption of local cache is a concern. If you're using a VM that [doesn't have a temporary disk](/azure/virtual-machines/azure-vms-no-temp-disk) then it's recommended to place `tempdb` on its own isolated disk or storage pool with caching set to read-only. To learn more, see [tempdb data caching policies](performance-guidelines-best-practices-storage.md#data-file-caching-policies).
```

File: azure-sql/virtual-machines/windows/performance-guidelines-best-practices-storage.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Many Azure VMs contain another disk type called the temporary disk (labeled as t
 
 The temporary storage drive isn't persisted to remote storage and therefore shouldn't store user database files, transaction log files, or anything that must be preserved.
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/performance-guidelines-best-practices-storage.md]:
```suggestion
The temporary storage drive isn't persisted to remote storage and therefore shouldn't store user database files, transaction log files, or anything that must be preserved. For example, you can use it for buffer pool extensions, the page file, and `tempdb`.
```

**Review by bluefooted — APPROVED** [review]

**Pull request merged**